app/src/main/java/com/worldsnas/starplayer/model/LocalMusic.kt

```diff
@@ -1,14 +1,10 @@
 package com.worldsnas.starplayer.model
 
-import android.os.Parcelable
-import kotlinx.android.parcel.Parcelize
-
-@Parcelize
```

Comment:
chera haz kardid ino?!
in model baraye ertebate 2 fragment bod faghat
age mikhayn esmesho ro behtar konin vali bayad Parcelable bashe

Reply:
daqiqan be dalile taqire nam